.expeditor/scripts/finish_release/notify_rustfmt.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+# Assumptions:
+# 1. This script runs from a bash action triggered from a workload subscription.
+#    Therefore it assumes EXPEDITOR_SCM_SETTINGS_BRANCH ENV var exists.
+
+# shellcheck source=.expeditor/scripts/shared.sh
+source .expeditor/scripts/shared.sh
+
+# https://expeditor.chef.io/docs/reference/workloads/#environment-variables
+BRANCH=${EXPEDITOR_SCM_SETTINGS_BRANCH?required EXPEDITOR_SCM_SETTINGS_BRANCH ENV var not set}
+
+export GITHUB_TOKEN
+GITHUB_TOKEN="${GITHUB_TOKEN:-$(chef_ci_github_token)}"
```

Comment:
I'm curious if we need these two lines. The `secrets` block in the yaml that fetches GITHUB_TOKEN  _should_ also set it in the environment.

Reply:
This runs in a Bash action in Expeditor, though; the `secrets` block is just for Buildkite.

However, if we're already grabbing from Vault, and the environment variable is scheduled to go away anyway, why don't we just rely on the Vault function directly and call it a day?

Reply:
:man_facepalming: You're right, I thought I was looking at the `bump_rustfmt.sh` script.

My understanding was that at some point in the future it won't be automatically present in the future for bash actions. I don't know what the replacement would look like, but I think we should prefer the secrets interface in the yaml to raw fetching when possible, as that will gives RelEng an abstraction to manage so that we won't need to update our scripts if they need to change, much like the change in how the queues for buildkite changed over time.

Reply:
@smacfarlane the Expeditor bash action docs actually call out that it's a preferred method to now use `vault` directly. https://expeditor.chef.io/docs/reference/actions/

>  Default access to these environment variables has been deprecated and will be removed in January 2020. Please shift to leveraging the new helpers/actions, or accessing the secret from Vault directly via the included vault CLI.

Reply:
:+1: Got it, guess it's time to re-read those docs :joy: